test/rest_api/rest_util.py

```diff
@@ -3,6 +3,8 @@
 
 from contextlib import contextmanager
 
+long_time = 1000000000
+
```

Comment:
I don't think this belongs to rest_util.
Just define it locally in the function that uses it.
There's nothing special in this constant.